care/emr/resources/patient/spec.py

```
@@ -47,12 +52,34 @@
 
 
+def validate_identifier_config(config, value):
+    if (
+        config["config"]["unique"]
+        and PatientIdentifier.objects.filter(
+            config__external_id=config["id"],
+            value=value,
+        ).exists()
+    ):
+        raise ValueError(
+            f"Identifier config {config['config']['system']} is not unique"
```

Ruff (0.11.9)

Exception must not use an f-string literal, assign to variable first

Assign to variable; remove f-string literal

(EM102)